custom_components/multiroom-mqtt-mediaplayer/media_player.py

```diff
@@ -271,6 +283,8 @@ def __init__(self, hass, config):
         self._supported_features = 0
 
         self._volume = 0.0
+        self._minvolume = -80.0
```

Comment:
Would be better not to change default behavior. Default would be from minvolume = 0 to maxvolume = 100

Reply:
Ah yes, thank you for checking! It’s a left over from my first version with static values from my setup